core/xim/xim.lua

```diff
@@ -1,11 +1,12 @@
 import("CmdProcessor")
 
 -- adjust the input parameters -> target arg1 arg2 arg3 ...
-function _input_params_process(target, ...)
-    local args = {...}
+function _input_params_process(target, args)
+
     if target:sub(1, 1) == '-' then
```

Comment:
function _input_params_process(target, ...)
    local args = {...}
    if target:sub(1, 1) == '-' then
        local original_target = target
        for i, arg in ipairs(args) do
            if arg:sub(1, 1) ~= '-' then
                target = arg
                table.remove(args, i)
                table.insert(args, 1, original_target)
                break
            end
        end
    end
    return target, args
end